_posts/2017-09-14-survey-results.md

```diff
@@ -0,0 +1,218 @@
+---
+title: The Pa11y User Survey 2017 results
+description: We ran the survey from August 15th to September 11th. Here's what we learned about you!
+date: 2017-12-08
+---
+
+We wanted to find out more about how our users use our tools, what sort of things people think we're doing well, and what sort of things we could do better. So we decided to run a survey to learn how you feel about the four main Pa11y packages. 
+
+We ran the survey from August 15th to September 11th. Here's what we learned about you!
+
+## Table of contents
+
+* [About our users](#about-our-users)
+* [Pa11y CLI](#pa11y-cli)
+* [Pa11y Dashboard](#pa11y-dashboard)
+* [Pa11y Webservice](#pa11y-webservice)
+* [Pa11y CI](#pa11y-ci)
+* [What we plan to improve](#what-we-plan-to-improve)
+	* [Documentation, tutorials, and alternatives](#documentation-tutorials-and-alternatives)
+	* [Code changes](#code-changes)
+* [Conclusions](#conclusions)
+
+
+## About our users
+
+More than three quarters of our respondents are developers. 
+
+![Pie chart: Do you mainly see yourself as a... Developer 72%; QA tester 5%; Accessibility tester 6%; UX specialist 6%; Other 11%](asset/image/2017-survey/1-job-type.png)
```

Comment:
Currently images don't load when you run Jekyll. I think the best thing to do would be to move your `2017-survey` folder of images into the root-level `asset/image`. Then update all the references in this file to be absolute

Reply:
probably should have actually checked that before committing 😬